torch/_dynamo/output_graph.py

```diff
@@ -609,14 +609,10 @@ def register_leaf_name(leaf_name):
                     # annoying, but there are cases when we do not have parameters
                     # see test_nn_moduledict_contains
                     if hasattr(target, "_parameters"):
-                        for leaf_name, _ in target.named_parameters(
-                            remove_duplicate=False
-                        ):
+                        for leaf_name, _ in target.named_parameters():
```

Comment:
@voznesenskym Is there any reason for remove_duplicate to be `False`?

Reply:
IIRC If you remove duplicates, you will get incomplete leaf names for AOTAutograd, and therefore name mismatches.